doc/stages/filters.trajectory.rst

```diff
@@ -0,0 +1,52 @@
+.. _filters.trajectory:
+
+filters.trajectory
+==================
+
+The **trajectory filter** computes an estimate the the sensor location based
+on the position of multiple returns and the sensor scan angle. It is primarily
+useful for LAS input as it requires scan angle and return counts in order to
+work.
+
+The method is described in detail `here`_. It extends the method of :cite:`Gatziolis2019`.
+
+.. note::
+
+This filter creates a new dataset describing the trajectory of the sensor,
+replacing the input dataset.
+
+Examples
+--------
+
+.. code-block:: json
+
+  [
+      "input.las",
+      {
+          "type": "filters.trajectory"
+      },
+      "trajectory.las"
```

Comment:
We can write an LAS, but I wonder if demonstrating the SBET writer might be more relevant in this case?